Client/Frontend/Browser/Playlist/Cells/PlaylistCellRedacted.swift

```diff
@@ -0,0 +1,225 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import CoreData
+import Shared
+import BraveUI
+import DesignSystem
+import BraveShared
+import Data
+
+@available(iOS, deprecated: 15.0, renamed: "SwiftUI.AsyncImage")
+public struct AsyncImage<Content: View>: View {
```

Comment:
nit: should this belong to it's own file? Doesn't seem to be used yet so maybe makes sense to keep here for it's intended purpose, but seems to just be backporting `AsyncImage` and it's initializers to < iOS 15